tests/pipfile_lock.test.ts

```diff
@@ -0,0 +1,32 @@
+jest.mock("danger", () => jest.fn())
+import * as danger from "danger"
+const dm = danger as any
+
+import { pipfileLock } from "../rules/python-prs"
+
+beforeEach(() => {
+  dm.danger = {}
+  dm.warn = jest.fn()
+})
+
```

Comment:
Maybe we could add a test case which does not warn when none of them was modified. What do you think? Does that make sense?

Reply:
Sure, that make sense.